runtime/integration-tests/src/pools/utils/env.rs

```diff
@@ -837,3 +841,41 @@ pub fn pass_n(env: &mut TestEnv, n: u64) -> Result<(), ()> {
 
 	Ok(())
 }
+
+fn default_parachains_host_configuration() -> HostConfiguration<BlockNumber> {
```

Comment:
Wondering whether we cannot get this from the relay chain spec which we use. This configuration is assumed to change over time.